monitoring/prometheus/prometheus_staging.yml

```diff
@@ -21,4 +22,4 @@ scrape_configs:
   - job_name: 'push_gateway'
     metrics_path: '/push_gateway/metrics'
     static_configs:
-      - targets: ['staging.eval.ai']
+      - targets: ['push_gateway:9091']
```

Comment:
This is not correct. If we are moving to separate nginx we should avoid referencing by container names. This should be a valid endpoint dns